docs/crd_mysql_backup_schedule.md

```diff
@@ -0,0 +1,44 @@
+# BackupSchedule
+
+`BackupSchedule` is a custom resource definition (CRD) that represents
+a full dump & binlog scheduling.
+
+`BackupSchedule` creates `CronJob` which collects binlog and then rotate dump.
+
+| Field        | Type                                          | Description                                          |
+| ------------ | --------------------------------------------- | ---------------------------------------------------- |
+| `apiVersion` | string                                        | APIVersion.                                          |
+| `kind`       | string                                        | Kind.                                                |
+| `metadata`   | [ObjectMeta]                                  | Standard object's metadata.                          |
+| `spec`       | [BackupScheduleSpec](#BackupScheduleSpec)     | Specification of scheduling.                         |
+| `status`     | [BackupScheduleStatus](#BackupScheduleStatus) | Most recently observed status of the scheduled jobs. |
+
+## BackupScheduleSpec
+
+| Field                    | Type                                    | Required | Description                                                               |
+| ------------------------ | --------------------------------------- | -------- | ------------------------------------------------------------------------- |
+| `clusterName`            | string                                  | Yes      | Name of [`MySQLCluster`](crd_mysql_cluster.md)                            |
+| `schedule`               | string                                  | Yes      | Schedule in Cron format, this value is passed to `CronJob.spec.schedule`. |
+| `objectStorageEndpoint`  | [ObjectStorageSpec](#ObjectStorageSpec) | Yes      | Specification of S3 compatible object storage.                            |
```

Comment:
should be changed to `objectStorageName` and type string.

Reply:
Fixed.